generate_constants.py

```diff
@@ -164,7 +164,7 @@
                               constNames=constNames,
                               upperValueString=upperValueString))
 
-test_header = """package ofxgo_test
+test_header = """package ofxgo
 
 /*
  * Do not edit this file by hand. It is auto-generated by calling `go generate`.
```

Comment:
Sorry I wasn't more explicit earlier. I believe `test_header` also needs to have the `"github.com/aclindsa/ofxgo"` import line removed, as well as several `ofxgo.` prefixes in `test_template`.